runner/status.go

```diff
@@ -151,3 +151,12 @@ func PreparingStatus(runID RunID) (r RunStatus) {
 	r.State = PREPARING
 	return r
 }
+
+// This is for overall runner status, just 'initialized' status for now.
+type ServiceStatus struct {
```

Comment:
I think it's fine this way until we have more specific needs (and we already track if a node is idle, though I guess there's some value to making that check explicit).